lib/std/multi_array_list.zig

```diff
@@ -391,13 +388,23 @@ pub fn MultiArrayList(comptime T: type) type {
             self.* = other;
         }
 
+        pub fn clearAndFree(self: *Self, gpa: Allocator) void {
+            gpa.free(self.allocatedBytes());
+            self.* = .{};
```

Comment:
```suggestion
            self.* = .empty;
```

Reply:
Ah, of course. Well, it will at the latest be fixed when the field defaults are removed, after 0.14.0 is tagged.